### ISSUE TITLE
List issues that dont need help anymore

### ISSUE THREAD

**Issue by ondrae** [codeforamerica/gotissues#7]
#### Description

There are a bunch of open github issues that are really old. They most likely don't need any help any more. Lets come up with a way to find issues that dont actually need help any more. Age is probably just the first step.

Start by listing out old issues. We can look at them and see which ones are still valid or not. If we can find something else common with them, then we can easily identify dead issues.

Then we can decide what to do with them. 
- Manually ping them asking if the issue is still relevant?
- Make a bot to ask if the issue is still relevant?
- Hide them from the Civic Issue Finder

**Comment by ondrae** [comment]
What is the cut off for an old issue? Created date or last updated date?

https://github.com/GSA/data.gov/issues/100 was created over a year ago, but had the "help wanted" label added within a year. We've [pinged them](https://github.com/GSA/data.gov/issues/100#issuecomment-119002452) to see if it still needs help.

[This issue](https://github.com/CodeForPortland/ideas/issues/10) is as old as the Civic Issue Finder. We've pinged them to see what happens too.

In this [highly clicked issue](https://github.com/Code-for-Miami/TwitterBot/issues/2) we pinged them and they removed the "help wanted" label.

**Comment by ondrae** [comment]
More evidence that pinging them works.
https://github.com/codefordenver/CodeAcross-2015/issues/12

**Comment by eunicekokor** [comment]
re: If we wanted to make a bot, [Github says](https://help.github.com/articles/what-s-the-difference-between-user-and-organization-accounts/) that we can:

```
User accounts are intended for humans, but you can give one to a robot, such as 
a continuous integration bot, if necessary.
```

I think now selecting those issues that we want is now the task. Tested a bot [here](https://github.com/codeforamerica/gotissues/issues/36). Currently the query I'm using is 
`q = ''' SELECT html_url,clicks,view_sources FROM issues WHERE state='open' '''`
But we can also pull `created_at` and see how long the issue has been open and only ping issues that are >30 days old.

This is optional but we can also figure out which ones we've pinged and check if their status has changed? Which, one way can be to, check once daily and see if there are any new closed issues that day (check `closed_at`) or even see if the # of comments increased by more than 1 (that means in addition to our comment, someone else responded) and it can send us an email (or something along those lines) that lets us know there has been an update.
